code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -18,7 +20,59 @@
 /obj/item/clothing/mask/muzzle/attack_hand(mob/user as mob)
 	if(user.wear_mask == src && !user.IsAdvancedToolUser())
 		return 0
+	else if(security_lock && locked)
+		if(do_unlock(user))
+			visible_message("<span class='danger'>[user] unlocks their [src.name].</span>", \
+								"<span class='userdanger'>[user] unlocks their [src.name].</span>")
 	..()
+	return 1
+
+/obj/item/clothing/mask/muzzle/proc/do_unlock(mob/living/carbon/human/user)
+	var/unlock = FALSE
+	if(istype(user.get_inactive_hand(), /obj/item/weapon/card/emag))
+		security_lock = FALSE
+		to_chat(user, "<span class='warning'>The lock vibrates as the card forces its locking system open.</span>")
+		unlock = TRUE
+	else if(access_brig in user.get_access())
+		unlock = TRUE
+	if(unlock)
+		to_chat(user, "<span class='warning'>The muzzle unlocks with a click.</span>")
+		locked = FALSE
+		flags &= ~NODROP
+		return TRUE
+
+	to_chat(user, "<span class='warning'>You must be wearing a security ID card or have one in your inactive hand to remove the muzzle.</span>")
+	return FALSE
+
+/obj/item/clothing/mask/muzzle/proc/do_lock(mob/living/carbon/human/user)
+	if(security_lock)
+		locked = TRUE
+		flags |= NODROP
+		return TRUE
+	return FALSE
+
+/obj/item/clothing/mask/muzzle/Topic(href, href_list)
+	..()
+	if(href_list["locked"])
+		var/mob/living/carbon/wearer = locate(href_list["locked"])
+		var/success = 0
+		if(ishuman(usr))

```

Comment:
You should probably include a way for xenos to simply rip off locked muzzles- Unless you want the new meta for Sec to become locking muzzles onto their troops so that they can't be face-hugged. (And, yes, I did confirm- [NODROP items cannot be removed by huggers](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/living/carbon/alien/special/facehugger.dm#L141-L142))